pyproject.toml

```diff
@@ -12,3 +12,4 @@ requires = [
 
 [tool.pytest.ini_options]
```

Comment:
This is required to remove the warning:
```
  /home/user/Repos/kivy/kivy/tests/pyinstaller/test_pyinstaller.py:17: PytestUnknownMarkWarning: Unknown pytest.mark.incremental - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.incremental
```